src/clj/game/cards/operations.clj

```diff
@@ -291,6 +291,17 @@
                               :effect (effect (trash target))}
                             card nil)))}}
 
+   "Enhanced Login Protocol"
+   {:events {:run {:once :per-turn
+                   ;; TODO: Find more elegant way of seeing if its a run event
+                   :req (req (not (has-subtype? (get (get (get @state :run) :run-effect) :card) "Run")))
+                   :effect (effect (update! (assoc card :elp-fire true))
+                             (lose :runner :click 1))
```

Comment:
The indentation style is to align arguments after a line break, so this is missing 6 spaces.